Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -20,6 +20,9 @@
 #import "FBLPromises.h"
 #endif
 
+#import "FIRExceptionModel_Private.h"
```

Comment:
If we pass the FIRExceptionModel into FIRCLSExceptionRecord, then we won't need to access the private header.